css/app.css

```diff
@@ -1,3 +1,243 @@
+/* Reset */
+
+*,
+*::after,
+*::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+ul {
+  list-style: none;
+}
+
+/* Globals */
+
 body {
-  background-color: blue;
+  font-family: "Lato", sans-serif;
+  text-align: center;
+  line-height: 1.4;
+  width: 100vw;
+  height: 100vh;
+  max-width: 100%;
+  margin: 0 auto;
+}
+
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-template-areas:
+    "header header header header"
+    "main main main main"
+    "footer footer footer footer";
+}
+
+/* Header */
+
+header {
+  grid-area: header;
+  background-color: #fff;
+  background-image: url(../img/hero.jpg);
+  background-position: center bottom;
+  background-size: cover;
+  background-repeat: no-repeat;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  align-content: center;
+  height: 600px;
+}
+
+nav {
+  display: flex;
+  flex: 2;
+}
+
+.nav {
+  display: flex;
+  align-self: center;
+  justify-content: center;
+  background: rgb(123, 123, 123);
+  background:
+    linear-gradient(
+      180deg,
+      rgba(123, 123, 123, 1) 41%,
+      rgba(123, 123, 123, 1) 43%,
+      rgba(96, 96, 96, 1) 48%,
+      rgba(115, 115, 115, 1) 100%
+    );
+  border-radius: 5px;
+}
+
+.nav li {
+  padding: 6px 30px;
+  color: #fff;
+  font-size: small;
+  position: relative;
+}
+
+.nav a {
+  color: #fff;
+  text-decoration: none;
+}
+
+.nav li input {
+  font-size: small;
+  background-color: inherit;
+  padding: 3px 5px 3px 22px;
+  border-radius: 50px;
+  width: 110px;
+  border-top-color: #444;
+  border-top-width: 1px;
+  border-left-width: 0;
+  transition: all 0.4s ease-in-out;
+}
+
+.nav li input:focus + i {
+  color: #333;
+}
+
+.nav li input:focus {
+  width: 200px;
+  transition: all 0.4s ease-in-out;
+  border: none;
+  outline: none;
+  background-color: #fff;
+}
+
+.nav li label i.fa-search {
+  font-size: 10px;
+  position: absolute;
+  left: 22px;
+  top: 12px;
+}
+
+.nav li:not(:first-of-type) {
+  border-left: 1px solid #636363;
+}
+
+.nav li:first-child {
+  font-size: 20px;
+  padding: 2px 30px;
+}
+
+.nav li:last-child {
+  padding: 6px 15px;
+}
+
+.footer-details .footer-links li {
+  padding: 0 10px;
+  line-height: 14px;
+  height: 14px;
```

Comment:
## Inneficient selectors[1]

Just keep with `.footer-links-li`. 
Look for all inefficient selectors in this file and then change them to avoid asking change just this in the next review.

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_